README.md

```diff
@@ -426,6 +426,11 @@ To show the domain names in search results instead of the expanded URL (and use
 
 3. You can find a rofi script for `googler` [here](http://hastebin.com/fonowacija.bash). Written by an anonymous user, untested and we don't maintain it.
 
+4. The [AlbertLauncher](https://albertlauncher.github.io) python plugins repo
```

Comment:
Please remove the link to Albert Launcher. It's fairly well-known and the repo links to it in the first line.

Reply:
Sure, done.